Readme.md

```diff
@@ -51,15 +51,30 @@ bundled with Dolphin and used if they're not installed on your system. CMake
 will inform you if a bundled library is used or if you need to install any
 missing packages yourself.
 
-### Build Steps:
-1. `mkdir Build`
-2. `cd Build`
+### OS X Build Steps:
+1. `mkdir build`
+2. `cd build`
 3. `cmake ..`
 4. `make`
 
-On OS X, an application bundle will be created in `./Binaries`.
+An application bundle will be created in `./Binaries`.
+
+### Linux Global Build Steps:
+
+The global build is recommended.
+
+1. `mkdir build`
+2. `cd build`
+3. `cmake ..`
+4. `make`
+5. `sudo make install`
 
-On Linux, it's strongly recommended to perform a global installation via `sudo make install`.
+### Linux Local Build Steps:
+1. `mkdir Build`
+2. `cd Build`
+3. `cmake .. -DLINUX_LOCAL_DEV=true`
+4. `make`
+5. `cp -r ../Data/Sys/ Binaries/`
```

Comment:
Maybe do a symlink instead, so next builds won’t require this step anymore (or worse, the user would forget it).